js/scripts.js

```diff
@@ -0,0 +1,18 @@
+﻿alert('(a * a) + (2 * a * b) - (b * b)');
+var a = prompt('podaj wartość a');
+var b = prompt('podaj wartość b');
+var value = (a * a) + (2 * a * b) - (b * b);
+alert(value);
+
+console.log('wynik działania ' +  value);
+
+if ( value > 0 ) {
+	console.log('wynik dodatni');
+} else if ( value < 0 ) {
+	console.log('wynik ujemny');
+} 
+else {
+	console.log('zero');
+}
+
```

Comment:
słuchaj zrobiłem z tego ternary operator, ale nie wiem co miałbym zrobić żeby wyszedł jeden if/else bo tam jest jeszcze 3 warunek "zero", no nie wiedziałem jak to ugryźć.